### PR TITLE
Feat/setup swagger

### PR DESCRIPTION
**Pull request by HristoSLV**
Setup swagger UI

### CODE REVIEW

**Review by ignatIgnatov — APPROVED** [review]
Looks Good!

File: build.gradle [hunk 1]
```diff
@@ -4,6 +4,7 @@ plugins {
 	id 'io.spring.dependency-management' version '1.1.6'
 	id 'jacoco'
 	id "org.sonarqube" version "4.4.1.3373"
+	//id "io.swagger.core.v3.swagger-gradle-plugin" version "2.2.24"
```
Comment (emiliyank) [inline build.gradle:7]: remove unused commented code

File: build.gradle [hunk 2]
```diff
@@ -31,6 +32,9 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 //	implementation 'org.liquibase:liquibase-core'
+	// https://mvnrepository.com/artifact/org.springdoc/springdoc-openapi-starter-webmvc-ui
```
Comment (emiliyank) [inline build.gradle:35]: remove commented code

**Review by emiliyank — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (HristoSLV) [reply build.gradle]: removed commented code

**Pull request merged**